site/using-weave/service-management.md

```diff
@@ -106,6 +107,25 @@ Furthermore, as explained in Binding Services, service locations can be dynamica
 For example, you can move the netcat service to `$HOST4:2211`  and it will retain its 10.2.1.3:3322 endpoint on the Weave network.
 
 
+###<a name="export-route"></a>Exposing the Weave Network More Widely
+
+You can use Linux routing to give access to the Weave network from
+hosts that are not running Weave. First, ensure that the [IP
+subnet](site/how-it-works/ip-addresses.md) used by Weave does not
+clash with anything on those other hosts, then on one Weave host run:
+
+    weave expose
+
+Then on each non-Weave machine that you want to access the network,
+run:
+
+    ip route add <network-cidr> via <exposing-host>
+
+Where `<network-cidr>` gives the IP address range used by weave,
```

Comment:
On further thought, this doesn't have to map to the IPAM range, just to some subnet in use on Weave Net that you want to route.